features/admin.groups.v1/components/edit-group/edit-group-users.tsx

```diff
@@ -460,12 +478,29 @@ export const GroupUsersList: FunctionComponent<GroupUsersListProps> = (props: Gr
 
     return (
         <EmphasizedSegment padded="very" className="list-group-roles-section">
-            <Heading as="h4">
-                { t("groups:edit.users.heading") }
-            </Heading>
-            <Heading subHeading ellipsis as="h6">
-                { t("groups:edit.users.subHeading") }
-            </Heading>
+            <Box display="flex" direction="row" justifyContent="space-between">
+                <div>
+                    <Heading as="h4">
+                        { t("groups:edit.users.heading") }
+                    </Heading>
+                    <Heading subHeading ellipsis as="h6">
+                        { t("groups:edit.users.subHeading") }
+                    </Heading>
+                </div>
+
+                { !isReadOnly && groupUserList?.totalResults > 0 && (
+                    <PrimaryButton
+                        data-testid={
```

Comment:
refarin from adding `data-testid`

Reply:
This change is to relocate the existing primary button from inside the search bar to the right side of the heading and sub heading. Hence removing test id will affect the tests.

Instead specified a component id along with the test id.